sklearn/model_selection/_search.py

```diff
@@ -1411,7 +1411,26 @@ class RandomizedSearchCV(BaseSearchCV):
         A generator over parameter settings, constructed from
         param_distributions.
 
+
+    Examples
+    --------
+    >>> from sklearn import linear_model, datasets
+    >>> from sklearn.model_selection import RandomizeSearchCV
+    >>> from scipy.stats import  uniform
+    >>> iris = datasets.load_iris()
+    >>> logistic = linear_model.LogisticRegression()
+    >>> hyperparameters = dict(C=uniform(loc=0, scale=4),
+    ...                        penalty=['l1', 'l2'])
+    >>> clf = RandomizedSearchCV(logistic, hyperparameters)
```

Comment:
Adding a `random_state=0` here should make the best parameters found deterministic.